docs/RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Release Instruction
+
+Please follow the following steps to publish a new version extension.
+
+1. Update the `version` property in [package.json](../package.json) file.
+
+2. Update [CHANGELOG](../CHANGELOG.md) file to add the notable changes of this version.

```

Comment:
no date in [`CHANGELOG.md`](https://github.com/docascode/vscode-docs-build/blob/master/CHANGELOG.md)?

Reply:
Good point, Added

---
In reply to: [361413067](https://github.com/docascode/vscode-docs-build/pull/6#discussion_r361413067) [](ancestors = 361413067)